File: que.py
class PatientQueue:
    def _init_(self):
        self.queue = []
    
    def enqueue(self, patient):
        self.queue.append(patient)
    
    def dequeue(self):
        if not self.queue:
            return None
        return self.queue.pop(0)

class EmergencyStack:
    def _init_(self):
        self.stack = []
    
    def push(self, emergency_patient):
        self.stack.append(emergency_patient)
    
    def pop(self):
        if not self.stack:
            return None
        return self.stack.pop()

File: que.cpp
#include <iostream>
#include <queue>
#include <stack>

class PatientQueue {
public:
    void enqueue(int patient) {
        queue.push(patient);
    }
    
    int dequeue() {
        if (queue.empty()) {
            return -1; // Assuming -1 indicates an empty queue
        }
        int patient = queue.front();
        queue.pop();
        return patient;
    }
private:
    std::queue<int> queue;
};

class EmergencyStack {
public:
    void push(int emergency_patient) {
        stack.push(emergency_patient);
    }
    
    int pop() {
        if (stack.empty()) {
            return -1; // Assuming -1 indicates an empty stack
        }
        int emergency_patient = stack.top();
        stack.pop();
        return emergency_patient;
    }
private:
    std::stack<int> stack;
};